src/main/java/io/confluent/connect/elasticsearch/jest/JestElasticsearchClient.java

```diff
@@ -193,6 +199,52 @@ public static HttpClientConfig getClientConfig(ElasticsearchSinkConnectorConfig
     return builder.build();
   }
 
+  private static void configureProxy(ElasticsearchSinkConnectorConfig config,
+      HttpClientConfig.Builder builder) {
+
+    if (config.isBasicProxyConfigured()) {
+      HttpHost proxy = new HttpHost(
+          config.getString(ElasticsearchSinkConnectorConfig.PROXY_HOST_CONFIG),
+          config.getInt(ElasticsearchSinkConnectorConfig.PROXY_PORT_CONFIG)
```

Comment:
is this the general style of the ES connector? or can you make getter methods?

Reply:
general style - altho maybe time to slowly start moving the style over